wpeverest/user-registration
# Dev-Change date() to date_i18n() function and Timezone Compatibility

### All Submissions:


### Changes proposed in this Pull Request:

This PR changes the default date() function to WordPress recommended date_i18n() function. Also, Make User-Registered Datetime as Timezone compatible as set by admin in settings.

### How to test the changes in this Pull Request:

1. Set Timezone in Dashboard->settings
2. Register a new user and check the user_registered column in user's section. There must be datetime as set timezone.
3. Test Date Field Completely. Maybe it can cause some issues there.

### Types of changes:




### Other information:




### Changelog entry

> Dev - Change date() to date_i18n() function and Timezone Compatibility
